lib/ButtonNew/ButtonIcon/ButtonIconDark.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import cx from 'classnames';
+import { Icon } from 'lib';
+import { ButtonBase } from '../ButtonBase';
+import {
+  sizes,
+  getSizeClasses,
+  buttonIconPropTypes,
+  buttonIconDefaultProps
+} from '../common';
+
+function ButtonIconDark({
+  className,
+  disabled,
+  name,
+  active,
+  size,
+  iconTitle,
+  children,
+  ...rest
+}) {
+  const nonDisabledClasses = cx(
+    'bg-transparent border-transparent focus:bg-neutral-40 focus:border-white focus:shadow-button-icon-dark-focus',
+    {
+      'active:bg-neutral-40 active:border-neutral-40 active:shadow-none hover:bg-neutral-30 hover:border-neutral-30': !active,
+      'button__icon-active bg-neutral-40 border-neutral-40 hover:bg-neutral-30': active
+    }
+  );
+
+  const classes = cx(
+    className,
+    'button__icon',
```

Comment:
What does this do? 🤔  (can tailwind & BEM live together? I'm sure you just copied this though so maybe there isn't an answer)

Reply:
Copied! 

Depending on what its doing I don't see a problem with this. I also quite like adding semantic/bem classes so I can see what is what when scanning the dom.

Reply:
I agree with adding semantic classes but maybe not BEM just because it confuses what we use to do with how we're trying to do things now. I've been adding semantic classes in hyphen format e.g. `button-icon-dark` which as you say helps identify them in the browser 👍 

I imagine this is doing something, making the component even more difficult to work with. Your call if you wanna make `ButtonIcon` easier to work with.